tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language_script.tsv

```diff
@@ -633,16 +635,17 @@ pdt	Plautdietsch	primary	Latn	Latin
 peo	Old Persian	secondary	Xpeo	Old Persian
 pfl	Palatine German	primary	Latn	Latin
 phn	Phoenician	secondary	Phnx	Phoenician
+pi	Pali	primary	Mymr	Myanmar
 pi	Pali	secondary	Deva	Devanagari
 pi	Pali	secondary	Sinh	Sinhala
 pi	Pali	secondary	Thai	Thai
 pis	Pijin	primary	Latn	Latin
 pko	Pökoot	primary	Latn	Latin
 pl	Polish	primary	Latn	Latin
 pms	Piedmontese	primary	Latn	Latin
-pnt	Pontic	primary	Cyrl	Cyrillic
 pnt	Pontic	primary	Grek	Greek
-pnt	Pontic	primary	Latn	Latin
+pnt	Pontic	secondary	Cyrl	Cyrillic
+pnt	Pontic	secondary	Latn	Latin
```

Comment:
What's the basis of making these secondary?

Reply:
Promoting Grek to be the primary script for Pontic.

Really for all current Pontic speakers its Grek in Greece, Latn in Turkey, and Cyrl in Russia/Ukraine. Pontic is only spoken by very marginal populations in Turkey and Russia, but it's a large recognized community in Greece.

What's the basis for primary v secondary?

Reply:
The primary vs secondary should be based on the literate population sizes. I forget what the cutoff is, but clearly if >50% of the usage of the language is in a particular script, that would be primary, not secondary. (But again, there might be bug in the code.)